website/src/examples/table-of-contents.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
Maybe we could lift this file up to the root directory? Keep config as flat and easily discoverable as possible.

Reply:
The paths defined in this file directly maps to the file names in the same directory. I find it pretty self-explanatory when they are in the same place.